babel.config.json

```diff
@@ -2,8 +2,6 @@
     "presets": ["@babel/preset-env", "@babel/preset-react"],
     "plugins": [
         ["@babel/plugin-proposal-decorators", {"legacy": true}],
-        "@babel/plugin-proposal-object-rest-spread",
-        "@babel/plugin-transform-flow-strip-types",
-        ["@babel/plugin-proposal-class-properties", {"loose": true}]
```

Comment:
i removed `{"loose": true}` to resolve the warning and tested the application a bit. i have not found any problems with removing `{"loose": true}` while testing.

if we remove `{"loose": true}`, we can remove the whole line because `@babel/plugin-proposal-class-properties` is included in `@babel/preset-env`